ValveResourceFormat/IO/FileExtract.cs

```diff
@@ -86,6 +117,14 @@ public static string GetExtension(Resource resource)
                 case ResourceType.PanoramaVectorGraphic: return "svg";
                 case ResourceType.Texture: return "png";
 
+                case ResourceType.Mesh:
+                case ResourceType.Model:
+                case ResourceType.World:
+                case ResourceType.WorldNode:
+                {
+                    return "gltf";
```

Comment:
Should probably use glb for the time being, gltf has some issues with huge files.